pkgs/applications/misc/quicksynergy/default.nix

```diff
@@ -0,0 +1,37 @@
+{stdenv, fetchurl, pkgs}:
```

Comment:
btw, so how do you guys know, which argument is "pkgs" and which is nix function (let's say, you are new person and don't know, grepping is the best option here? any other way to find what is what?)